circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
```

Comment:
I think if we begin to actively work on the app in this branch then we should turn on tests, but until then there's no need.